packages/titanium-data-table/src/titanium-data-table.ts

```diff
@@ -202,7 +202,8 @@ export class TitaniumDataTableElement extends LitElement {
   }
 
   private _determineTake() {
-    const take = Number(window.localStorage.getItem(`${this.header}-take`)) || 0;
+    const id = this.getAttribute('id');
```

Comment:
Confused a bit here.  There isn't a Id attribute and we don't set one on data tables.

Reply:
Id is a standard html attribute that can be set on any dom element. I'm just looking for a way to give tables an identifier in the case where header isn't static... ("totally unrelated", but I'm making the header for the logs table static...)